src/test/java/com/exasol/adapter/dialects/exasol/AbstractExasolSqlDialectIT.java

```diff
@@ -89,6 +111,17 @@ private static AdapterScript installVirtualSchemaAdapter(final ExasolSchema adap
                 .build();
     }
 
+    protected static boolean exasolVersionSupportsFingerprintInAddress() {
+        final ExasolDockerImageReference imageReference = EXASOL.getDockerImageReference();
+        if (imageReference.getMajor() <= 6) {
+            return false;
+        }
+        if (imageReference.getMinor() == 0) {
+            return false;
```

Comment:
Maybe I would do it with `imageReference.getMajor() < 7` and `imageReference.getMinor() < 1` just to make it more explicit that a fingerprint is supported from 7.1.

Reply:
✅ 
```java
return (imageReference.getMajor() >= 7) && (imageReference.getMinor() >= 1);
```